ChiModules/CFEMDiffusion/lua/ds_lua_utils.h

```diff
@@ -0,0 +1,20 @@
+#ifndef CFEM_DIFFUSION_LUA_UTILS_H
+#define CFEM_DIFFUSION_LUA_UTILS_H
+
+#include"ChiLua/chi_lua.h"
+#include "../cfem_diffusion_solver.h"
+
+int chiCFEMDiffusionSolverCreate(lua_State *L);
+int chiCFEMDiffusionSetBCProperty(lua_State *L);
+
+
+namespace cfem_diffusion
```

Comment:
@Naktakala I blindly copied for LBS eigensolver. So I hve question: is this cfem_diffusion namespace the SAME namespace in in cfem_diffusion_solver.h ???